fjord/feedback/views.py

```diff
@@ -45,8 +46,15 @@ def download_firefox(request, template):
 
 
 def thanks(request):
-    template = 'feedback/thanks.html'
-    return render(request, template)
+    user_opinion = Response.objects.get(id=request.session['opinion_id'])
```

Comment:
If there's no "opinion_id" in the session, then this will throw an error. I think it's worth unpacking this line into a couple of lines so that we can do proper error checking/handling.